src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.Android.cs

```diff
@@ -51,7 +51,7 @@ public PointerPoint GetCurrentPoint(UIElement relativeTo)
 		{
 			var timestamp = ToTimeStamp(_nativeEvent.EventTime);
 			var device = PointerDevice.For(Pointer.PointerDeviceType);
-			var rawPosition = new Point(_nativeEvent.RawX, _nativeEvent.RawY); // Relative to the screen
+			var rawPosition = new Point(_nativeEvent.RawX, _nativeEvent.RawY).PhysicalToLogicalPixels(); // Relative to the screen
```

Comment:
Oooh

... is the comment still relevant?

Reply:
Yes :)